packages/core/rollup.config.js

```diff
@@ -29,47 +29,10 @@ const config = {
   baseDir: './src',
   outDir: './dist/core',
   entryPoints: {
-    modules: ['./src', './src/internal', './src/polyfills', './src/test'],
-    components: [
-      './src/accordion',
-      './src/alert',
-      './src/badge',
-      './src/breadcrumb',
-      './src/button',
-      './src/card',
-      './src/checkbox',
-      './src/datalist',
-      './src/date',
-      './src/divider',
-      './src/file',
-      './src/forms',
-      './src/icon',
-      './src/input',
-      './src/internal-components/close-button',
-      './src/internal-components/overlay',
-      './src/internal-components/panel',
-      './src/internal-components/visual-checkbox',
-      './src/modal',
-      './src/navigation',
-      './src/pagination',
-      './src/password',
-      './src/progress-circle',
-      './src/radio',
-      './src/range',
-      './src/search',
-      './src/select',
-      './src/selection-panels/checkbox',
-      './src/selection-panels/radio',
-      './src/tag',
-      './src/textarea',
-      './src/test-dropdown',
-      './src/time',
-      './src/toggle',
-      './src/tree-view',
-    ],
+    modules: ['./src/**/index.ts', './src/**/register.ts'],
+    sideEffects: ['./src/**/register.ts', './src/polyfills/*.ts'],
```

Comment:
Main cleanup with the rollup build is to use a glob instead of manually adding each entrypoint. Conventionally `index.ts` and `register.ts` files will be marked for compilation